src/components/Form.spec.js

```diff
@@ -0,0 +1,14 @@
+import { render, screen } from '@testing-library/react'
+import Form from './Form'
+
+describe('Form', () => {
```

Comment:
I would add a test where you test that the prop `onSubmit` is called with the form data.